server/container_create.go

```diff
@@ -1038,14 +1038,24 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 	}
 
 	// Get imageName and imageRef that are later requested in container status
-	status, err := s.StorageImageServer().ImageStatus(s.ImageContext(), images[0])
-	if err != nil {
-		return nil, err
+	var (
+		imgResult    *storage.ImageResult
+		imgResultErr error
+	)
+	for _, img := range images {
+		imgResult, imgResultErr = s.StorageImageServer().ImageStatus(s.ImageContext(), img)
+		if err == nil {
```

Comment:
Is this checking the right variable?

Reply:
whoops, I updated the line above it and overlooked the check. Fixed now.